extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusAst.js

```diff
@@ -26,7 +27,8 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusAst',[
 		};
 
 	function isEnabled() {
```

Comment:
I guess this function is used somewhere inside prebid, but anyway I will consider to extract it to `adContext.js` or all of prebid related flags to some "special" biddersAdContext